README.md

```diff
@@ -33,7 +30,8 @@ An `nvmrc` file is included if using [nvm](https://github.com/creationix/nvm).
 From within the root directory:
 
 ```sh
-npm install -g webpack
-npm install
+npm init
+npm install --save express dotenv
+npm install --save-dev nodemon eslint eslint-config-airbnb
```

Comment:
I'm not sure I understand this part.

It's all going to be in your `package.json`, so `npm install` would cover it, wouldn't it?

Reply:
You're right that I don't need these anymore. I was keeping track of what I was doing as I was setting up.  Thanks for the reminder. I've taken care of this now.